style.css

```diff
@@ -63,6 +63,14 @@ h1 {
   border-color: black;
 }
 
+.color-box:hover {
+  cursor: grab;
+}
+
+.color-box:active {
+  cursor: grabbing;
```

Comment:
I hope we can figure out how to make this grabbing cursor stick during the dragging phase